intg/src/main/java/org/apache/atlas/model/instance/AtlasRelationship.java

```diff
@@ -85,11 +88,13 @@ public class AtlasRelationship extends AtlasStruct implements Serializable {
     private Date          updateTime     = null;
     private Long          version        = 0L;
 
-    public enum Status { ACTIVE, DELETED }
+    public enum Status { ACTIVE, DELETED, PURGE_DUE_TO_ENTITY }
```

Comment:
Can you add details or comments in the code about what is `PURGE_DUE_TO_ENTITY`

Reply:
Done